### ISSUE TITLE
Repositioning the plot toolwindow with drag & drop caused VS hang

### ISSUE THREAD

**Issue by huguesv** [Microsoft/RTVS#2374]
It was stuck in the special transient UI that VS has for helping you re-arrange tool windows.

Happened on Shahrokh's machine. I put logs and dump here:
\\pytools\r\Bugs\plotwindowstuckdragdrop

**Comment by MikhailArkhipov** [comment]
@huguesv - anything we need to take to 0.5 from here?

**Comment by huguesv** [comment]
Not easy to repro, but possible.  I kept moving the toolwindow around, eventually it happened.  It looks like it's the drag & drop of plot themselves that interferes with the VS drag & drop of tool windows.

If I break into the debugger while it's stuck, I got:
```
 	[External Code]	
>	Microsoft.R.Components.dll!Microsoft.R.Components.Plots.Implementation.View.RPlotDeviceControl.Image_MouseMove(object sender, System.Windows.Input.MouseEventArgs e) Line 130	C#
 	[External Code]	
```

Stuck into this call:
```
DragDrop.DoDragDrop(this, obj, DragDropEffects.Copy | DragDropEffects.Move);
```

**Comment by huguesv** [comment]
![image](https://cloud.githubusercontent.com/assets/1696845/18798859/55edb97e-8189-11e6-9a24-96f45d81d92f.png)

**Comment by MikhailArkhipov** [comment]
I'd switch off just my code, enabled native and point symbols to `\\ddrps\symbols` and `\\symbols\symbols` - it is probably stuck somewhere in thread transitions

**Comment by huguesv** [comment]
I'm inclined to say we take the fix I have for 0.5, linked above.  Once you know how to repro, it's actually pretty easy:
- Left click inside plot window.
- Left click on the "R Plot" tool window label and start dragging **up** such that you'll send MouseMove events to plot window.
- Plot window incorrectly thinks you started dragging a plot, and now you are multi tasking drag & drop :)
